data/built-ins.json

```diff
@@ -100,26 +100,31 @@
   },
   "es6.map": {
     "chrome": 51,
+    "firefox": 53,
```

Comment:
again trying to be correct first/conservative and then we can optimize like in https://github.com/babel/babel-preset-env/pull/30